app/main.py

```diff
@@ -1,6 +1,32 @@
+import os
+
+import requests
+
+API_KEY = os.environ.get("API_KEY")
+URL = "https://api.weatherapi.com/v1/current.json"
+FILTERING = "Paris"
```

Comment:
The variable `FILTERING` is not a constant, because it represents a particular value that could change depending on the context. In Python, we usually use uppercase for constants that never change. In this case, it would be more appropriate to use a lowercase name, like `filtering`, to follow Python's naming conventions (PEP 8).